features/step_definitions/supplieraz_steps.rb

```diff
@@ -41,3 +41,54 @@
     ".//h2[@class='search-result-title']/a"
   ).text.should == normalize_whitespace(@supplier['name'])
 end
+
+Given(/I navigate to the list of '(.*)' page$/) do |value|
+  page.visit("#{dm_frontend_domain}/g-cloud/suppliers?prefix=#{value.split(' ').last}")
+  @data_store = @data_store || Hash.new
+  @data_store['supplier_alphabet'] = "#{value.split(' ').last}"
+end
+
+Then(/I am taken to page '(.*)' of results$/) do |page_number|
+  if current_url.include?('search?')
+    current_url.should include("#{dm_frontend_domain}/g-cloud/search?")
+    current_url.should include("page=#{page_number}")
+    current_url.should include("lot=iaas")
```

Comment:
I don't think checking the lot here is applicable? We do have `framework=g-cloud` in the url now though.